splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/PreparedStatementIT.java

```diff
@@ -625,4 +625,31 @@ public void testCursorUntypedExpressionType() throws Exception {
             methodWatcher.execute("call syscs_util.syscs_set_global_database_property('splice.bind.cursorUntypedExpressionType', null)");
         }
     }
+
+    @Test
+    public void testConcatUnknownParameters() throws Exception {
+        testParamsHelper("select ? || ?",
+                new Object[] {"abc", "def"},
```

Comment:
In IT there are tests only for strings. Does it make sense to add other types or the type casting is covered enough by other tests?